substrait_consumer/consumers/acero_consumer.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+import string
+from pathlib import Path
+from typing import Iterable
+
+import pyarrow as pa
+import pyarrow.parquet as pq
+import pyarrow.substrait as substrait
+
+from substrait_consumer.common import SubstraitUtils
+
+from .consumer import COLUMN_A, COLUMN_B, COLUMN_C, COLUMN_D, Consumer
+
+
+class AceroConsumer(Consumer):
+    """
+    Adapts the Acero Substrait consumer to the test framework.
```

Comment:
The words "test framework" feel off.  You are probing/validating behavior by running queries.  When I hear the word test I am thinking that you're testing this code which isn't the case.  Not worth changing at this point, however.